configs/zerow_defconfig

```diff
@@ -0,0 +1,63 @@
+BR2_TARGET_GENERIC_HOSTNAME="zero_w"
+BR2_TARGET_GENERIC_ISSUE="Welcome to the zero_w board"
+
+# board/compiler target setup
+BR2_aarch64=y
+BR2_cortex_a53=y
+
+# post build sdcard image generaton
+BR2_ROOTFS_POST_BUILD_SCRIPT="$(BR2_EXTERNAL)/board/S905Y2/post-build.sh"
+BR2_ROOTFS_POST_IMAGE_SCRIPT="$(BR2_EXTERNAL)/board/S905Y2/post-image.sh"
+
+# Filesystems
+BR2_TARGET_ROOTFS_EXT2=y
+BR2_TARGET_ROOTFS_EXT2_4=y
+BR2_TARGET_ROOTFS_EXT2_SIZE="250M"
+BR2_PACKAGE_HOST_DOSFSTOOLS=y
+BR2_PACKAGE_HOST_MTOOLS=y
+BR2_PACKAGE_HOST_UBOOT_TOOLS=y
+BR2_PACKAGE_HOST_GENIMAGE=y
+
+# u-boot
+BR2_TARGET_UBOOT=y
+BR2_TARGET_UBOOT_BUILD_SYSTEM_KCONFIG=y
+BR2_TARGET_UBOOT_USE_DEFCONFIG=y
+BR2_TARGET_UBOOT_CUSTOM_GIT=y
+BR2_TARGET_UBOOT_CUSTOM_REPO_URL="https://github.com/radxa/u-boot.git"
```

Comment:
I notice you didn't use the stock uboot - perhaps we can talk about that on signal. OK for now,

Reply:
I was looking into the radxa wiki:
https://wiki.radxa.com/Zero/dev/u-boot
And that's the repo they used in their example. So I took it from there.